scripts/lowercase-all-records.js

```diff
@@ -10,10 +10,9 @@ const getSha1 = require("../sha1-utils");
 
 
 async function subscribeLowercaseHashToHIBP(emailAddress) {
-  const lowerCasedEmail = emailAddress.toLowerCase();
-  const lowerCasedSha1 = getSha1(lowerCasedEmail);
-  await HIBP.subscribeHash(lowerCasedSha1);
-  return lowerCasedSha1;
+  const emailHash = getSha1(emailAddress);
+  await HIBP.subscribeHash(emailHash);
+  return emailHash;
```

Comment:
This bit feels interesting here. If I'm following the diffs correctly, previously this would have been a lowercased SHA, but now we're explicitly calling `.toUpperCase()` on the `return` statement of `getSha1Hash()` below. Not sure if there will be any fallout from changing the case here, but I suspect not. Especially since it looks like we call `.toLowerCase()` on the string again below on L28 and L42 when inserting the hashes into the DB.
Juggling the string cases feels a bit fragile. Not sure if it's easier to just always return a lowercased hash and then force the string to uppercase in the `HIBP.subscribeHash()` method, if "Hashes must be uppercased to return correct results from HIBP".